spec/models/comment_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+
+  describe 'default post details' do
+    let(:user) { create :user }
+    let(:second) { create :user }
+    let(:third) { create :user }
+    let(:post) { create :post }
+    let(:valid_comment) { build :comment}
+    let(:comment) { create :comment}
+
+    context "Comment validations :" do
+      it 'should be valid at creation ' do
+        expect(valid_comment.valid?).to be true
+      end
+
+      it 'should be invalid without body' do
+        valid_comment.body = nil
+        expect(valid_comment.valid?).to be false  
```

Comment:
It's better we have a separate variable `invalid_comment` to make this more intuitive:

`expect(invalid_comment).to_not be_valid`

Reply:
Okay, but we actually want to check if it is not valid without body, not overall. And it does not seem good to create a variable for every instance if we only use it 1 time

Reply:
Haha you are correct @YellowPipe! Great catch! :)